server/games/game.py

```diff
@@ -70,6 +72,7 @@ def __init__(
         enforce_rating_range: bool = False,
         max_players: int = 12,
         setup_timeout: int = 60,
+        hosted_at: datetime = datetime_now
```

Comment:
`datetime_now` is a function. You probably need to set the default to `None` and then do `self.hosted_at = hosted_at or datetime_now()`